autoPyTorch/pipeline/components/setup/network_backbone/ShapedResNetBackbone.py

```diff
@@ -188,9 +196,12 @@ def get_hyperparameter_search_space(  # type: ignore[override]
 
         if skip_connection_flag:
 
+            shake_shake_flag = False
             shake_drop_prob_flag = False
             if 'shake-drop' in multi_branch_choice.value_range:
                 shake_drop_prob_flag = True
+            elif 'shake-shake' in multi_branch_choice.value_range:
```

Comment:
Same in the L190--195 of `autoPyTorch/pipeline/components/setup/network_backbone/ResNetBackbone.py`